src/store/rootReducer.js

```diff
@@ -0,0 +1,9 @@
+import TODOListReducer from "./TODOListReducer";
```

Comment:
This code already exists in main in the index.js file

Reply:
Uh I changed it slightly. rootReducer.js now combines all the reducers with configureStore. In index.js, we now import the combined store (output of  configureStore) like this. Is this okay?

<img width="807" alt="Screen Shot 2023-06-05 at 2 39 01 PM" src="https://github.com/awctw/study-dash/assets/105681176/1af5aeca-390d-4048-8271-5615c8ee7cd1">

Reply:
This is exactly what the current store does in the main branch. This is the index.js file in store folder and in the sec folder index.js we are doing the same: import { configureStore } from "@reduxjs/toolkit";
import todoReducer from "./todo-slice";

const store = configureStore({
  reducer: { todoReducer },
});

export default store;

Reply:
So this is really a question of which store file we should go with. I recommend this one because the code for combined the redux stores are all placed in rootReducer rather than index.js. It feelts better organized? What do you think?

Reply:
It is the same approach no?, only difference is the name of the file -> index.js vs rootReducer.js from what I see. I said to use the original in main purely because you will have to delete something in main already replacing it with the same logic.

Reply:
But separating the store from index.js is better design right?

Reply:
If you want to have the combined redux store be in index.js, I can make that change.

Reply:
This is your rootReducer.js:
```
import TODOListReducer from "./TODOListReducer";
import {configureStore} from "@reduxjs/toolkit";

export const store = configureStore({
    reducer: {
        todoReducer: TODOListReducer
    },
    devTools: true
});
```
This is what is inside the index.js file in the store folder:

```
import { configureStore } from "@reduxjs/toolkit";
import todoReducer from "./todo-slice";

const store = configureStore({
  reducer: { todoReducer },
});

export default store;
```

No difference, the export is just following React customs for export and imports

Reply:
Uhh... the index.js in the store has been removed in my branch

Reply:
Can all of us just use the rootReducer to combine our various data slices?

Reply:
We can use it but its just weird that we are deleting code and then adding the exact same code back in with the only difference being index.js was renamed to rootReducer.

Reply:
Well. Having two files both named index.js is a bit confusing haha